package.json

```
@@ -16,7 +16,7 @@
   },
   "homepage": "https://github.com/dsb-norge/vue-keycloak-js",
   "dependencies": {
-    "keycloak-js": "4.8.3"
+    "keycloak-js": "8.0.0"
```

We stick to the version released in Red Hat Single Sign On for now, but users are free to use a newer version. We should document this, and how to include a more recent version...

```suggestion
    "keycloak-js": "4.8.3"
```